documentation/templates/documentation/map.html

```diff
@@ -0,0 +1,101 @@
+{% extends "cdo_base.html" %}
+
+{% load staticfiles %}
+{% load mezzanine_tags %}
+{% load keyword_tags %}
+
+{% block meta_keywords %}{% metablock %}
+    {% keywords_for map as keywords %}
+    {% for keyword in keywords %}
+        {% if not forloop.first %}, {% endif %}
+        {{ keyword }}
+    {% endfor %}
```

Comment:
is there a reason this isn't just a `join`?

Reply:
That's the pattern mezzanine uses everywhere else, the join template tag definitely would be cleaner